<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Password_change_windowClass</class>
 <widget class="QDialog" name="Password_change_windowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>275</width>
    <height>294</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Password_change_window</string>
  </property>
  <widget class="QWidget" name="widget_3" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>281</width>
     <height>291</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #EBE9F5;
border-radius: 20px;

</string>
   </property>
   <widget class="QLineEdit" name="current_password_input">
    <property name="geometry">
     <rect>
      <x>41</x>
      <y>80</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 11px;
border-width: 1px;
background-color: #FFFFFF;
color: #7F7F7F;
padding: 5px;</string>
    </property>
    <property name="placeholderText">
     <string>Current password</string>
    </property>
   </widget>
   <widget class="QToolButton" name="change_button">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>250</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: outset ; 
border-radius: 11px;
border-color:  #7D43E2;
border-width: 1px;
background-color: #7D43E2;
color: #FFFFFF;</string>
    </property>
    <property name="text">
     <string>Change</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="new_password_input">
    <property name="geometry">
     <rect>
      <x>41</x>
      <y>124</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Source Code Pro</family>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 11px;
border-width: 1px;
background-color: #FFFFFF;
color: #7F7F7F;
padding: 5px;</string>
    </property>
    <property name="placeholderText">
     <string>New password</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>16</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Change Passwrod</string>
    </property>
   </widget>
   <widget class="QLabel" name="password_status">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>170</y>
      <width>231</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>7</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:red</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
